src/main/java/org/threadly/litesockets/utils/MergedByteBuffers.java

```diff
@@ -17,13 +17,13 @@
  */
 public class MergedByteBuffers {
```

Comment:
Maybe your right.  I know a StringBuffer is thread safe, where a StringBuilder is not thread safe.  I thought they were trying to make "buffer" mean thread safe, but I could be mistaken.